server/routerlicious/packages/lambdas/src/deli/checkpointManager.ts

```diff
@@ -44,35 +45,47 @@ export interface ICheckpointParams {
 }
 
 export function createDeliCheckpointManagerFromCollection(
-	tenantId: string,
-	documentId: string,
-	collection: ICollection<IDocument>,
-): IDeliCheckpointManager {
-	const checkpointManager = {
-		writeCheckpoint: async (checkpoint: IDeliState) => {
-			return collection.update(
-				{
-					documentId,
-					tenantId,
-				},
-				{
-					deli: JSON.stringify(checkpoint),
-				},
-				null,
-			);
-		},
-		deleteCheckpoint: async () => {
-			return collection.update(
-				{
-					documentId,
-					tenantId,
-				},
-				{
-					deli: "",
-				},
-				null,
-			);
-		},
-	};
-	return checkpointManager;
+    tenantId: string,
+    documentId: string,
+    collection: ICollection<IDocument>,
+    localCollection: ICollection<ICheckpoint>): IDeliCheckpointManager {
+    const checkpointManager = {
+        writeCheckpoint: async (checkpoint: IDeliState, isLocal: boolean) => {
+            return isLocal ? localCollection.upsert({
+                documentId,
+                tenantId,
```

Comment:
Need to provide an `_id` with value of `documentId` in filter, this is needed for point operation, which will be huge benefits for us.